lex_node/launch/lex.launch.py

```diff
@@ -18,15 +18,30 @@
 from ament_index_python.packages import get_package_share_directory
 from launch import LaunchDescription
 from launch_ros.actions import Node
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
 
 
 def generate_launch_description():
-    # TODO(wjwwood): Use a substitution to find share directory once this is implemented in launch
-    parameters_file_path = os.path.join(get_package_share_directory('lex_node'),
+    config_file_path = os.path.join(get_package_share_directory('lex_node'),
                                         'config', 'sample_configuration.yaml')
     return LaunchDescription([
-        Node(package='lex_node',
-             node_executable='lex_node',
-             parameters=[parameters_file_path],
-             output='screen'),
+        DeclareLaunchArgument(
+            name='node_name',
+            default_value='lex_node'
+        ),
+        DeclareLaunchArgument(
+            name='config_file',
+            default_value=config_file_path
+        ),
+        Node(
+            package='lex_node',
+            node_executable='lex_node',
+            node_name=LaunchConfiguration('node_name'),
+            parameters=[LaunchConfiguration('config_file')],
+            output='screen',
+            additional_env={
+                'RCUTILS_CONSOLE_STDOUT_LINE_BUFFERED': '1'
```

Comment:
Why is this something we're adding to this Cloud extension and none of the others?

Reply:
Without it the Lex output was being buffered until the app was killed. Do the other cloud extensions have some other workaround for this?

Reply:
That makes sense, I'm not sure how it's working for the other CEs

Reply:
Have changed it to https://github.com/ros2/demos/blob/388a07d96286f6a027e7403a5eac2f618257dae3/logging_demo/src/logging_demo_main.cpp#L24 which will be more reliable (so will work using other launch files etc).